### ISSUE TITLE
Generator fails: error  "swagger-express-middleware" is not found, when selecting OpenAPI 3

### ISSUE THREAD

**Issue by konst-gkinis** [cdimascio/generator-express-no-stress#42]
node -v : 12.13.0
npm -v: 6.12.0

Tried both with -g install and npx.

Same error for node 13.1.0

### **The issue only happens when selecting Open API 3**

Using the generator fails with:

```
<project path>/server/common/swagger.js
  1:24  error  "swagger-express-middleware" is not found  node/no-missing-import
✖ 1 problem (1 error, 0 warnings)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! site@1.0.0 lint:fix: `eslint --fix -c .eslintrc.json {server,test}/**`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the site@1.0.0 lint:fix script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     <log path>
```

Here is the debug.log file contents:

```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/Users/kg/.nvm/versions/node/v12.13.0/bin/node',
1 verbose cli   '/Users/kg/.nvm/versions/node/v12.13.0/bin/npm',
1 verbose cli   'run',
1 verbose cli   'lint:fix'
1 verbose cli ]
2 info using npm@6.12.0
3 info using node@v12.13.0
4 verbose run-script [ 'prelint:fix', 'lint:fix', 'postlint:fix' ]
5 info lifecycle site@1.0.0~prelint:fix: site@1.0.0
6 info lifecycle site@1.0.0~lint:fix: site@1.0.0
7 verbose lifecycle site@1.0.0~lint:fix: unsafe-perm in lifecycle true
8 verbose lifecycle site@1.0.0~lint:fix: PATH: /Users/kg/.nvm/versions/node/v12.13.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/kg/Desktop/rendezvous/site/node_modules/.bin:/Users/kg/.npm/_npx/4647/bin:/Users/kg/Desktop/rendezvous/node_modules/.bin:/Users/kg/.nvm/versions/node/v12.13.0/bin:/Users/kg/.nvm/versions/node/v13.1.0/bin:/Users/kg/.nvm/versions/node/v13.1.0/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/usr/local/share/dotnet:~/.dotnet/tools:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/kg/.nvm/versions/node/v13.1.0/bin
9 verbose lifecycle site@1.0.0~lint:fix: CWD: /Users/kg/Desktop/rendezvous/site
10 silly lifecycle site@1.0.0~lint:fix: Args: [ '-c', 'eslint --fix -c .eslintrc.json {server,test}/**' ]
11 silly lifecycle site@1.0.0~lint:fix: Returned: code: 1  signal: null
12 info lifecycle site@1.0.0~lint:fix: Failed to exec lint:fix script
13 verbose stack Error: site@1.0.0 lint:fix: `eslint --fix -c .eslintrc.json {server,test}/**`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/Users/kg/.nvm/versions/node/v12.13.0/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:210:5)
13 verbose stack     at ChildProcess.<anonymous> (/Users/kg/.nvm/versions/node/v12.13.0/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:210:5)
13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
14 verbose pkgid site@1.0.0
15 verbose cwd /Users/kg/Desktop/rendezvous/site
16 verbose Darwin 19.0.0
17 verbose argv "/Users/kg/.nvm/versions/node/v12.13.0/bin/node" "/Users/kg/.nvm/versions/node/v12.13.0/bin/npm" "run" "lint:fix"
18 verbose node v12.13.0
19 verbose npm  v6.12.0
20 error code ELIFECYCLE
21 error errno 1
22 error site@1.0.0 lint:fix: `eslint --fix -c .eslintrc.json {server,test}/**`
22 error Exit status 1
23 error Failed at the site@1.0.0 lint:fix script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by cdimascio** [comment]
Thanks for the issue, @raidenfreeman . Will have a look shortly

**Comment by cdimascio** [comment]
@raidenfreeman this is resolved in v8.6.4. please give it try. thanks again

**Comment by cdimascio** [comment]
fixed for real in `v8.7.0`

**Comment by blackbasilisk** [comment]
Node v18.13.0
npm v9.19.3

I just followed the process to install. Selected OpenAPI 3 and Prettier as the linter and it gives me this error. Followed the steps exactly like they are in the Readme. I assume "npm install -g yo generator-express-no-stress" installs the latest version which should have this bug fixed right?
